backend/app/tests/conftest.py

```
@@ -16,6 +17,11 @@
 from app.seed_data.seed_data import seed_database
 
 
+@pytest.fixture(scope="session", autouse=True)
+def load_test_env():
+    load_environment("../.env.test")
```

Ruff (0.12.2)

Undefined name `load_environment`

(F821)